src/app/(post-login)/submission/layout.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+'use client';
+
+import { HEADER_HEIGHT_PX } from '@/app/components/Header';
+import { css, useTheme } from '@/lib/emotion';
+import mockData from '@/mockData.json';
+import { ReactNode, useState } from 'react';
+import SideMenu from './components/SideMenu/Menu';
+import TitleBar from './components/TitleBar';
+
+export default function AppLayout({ children }: { children: ReactNode }) {
+	const theme = useTheme();
+	const programData = mockData.programs;
+	const [isSidebarActive, setSidebarActive] = useState<boolean>(true);
+
+	return (
+		<div
+			css={css`
+				display: grid;
+				grid-template-columns: ${isSidebarActive ? '248px' : '40px'} 1fr;
```

Comment:
Also not for this PR, but the same reusable variables treatment could be used here.